styles/styles.css

```diff
@@ -0,0 +1,652 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A;
+    --white: #FFFFFF;
+}
+
+body {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-weight: 300;
+    font-size: 20px;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+h1, h2, h3, h4, h5 {
+    font-family: Merriweather, 'Times New Roman', serif;
+}
+
+h1 {
+    font-size: 36px;
+    padding: 16px 0;
+    margin-bottom: 10px;
+
+}
+
+h2 {
+    font-size: 36px;
+    padding: 16px 0;
+}
+
+h3 {
+    font-size: 24px;
+    margin-bottom: 15px;
+}
+
+h4 {
+    font-size: 22px;
+    margin: 6px 0;
+    font-weight: 200;
+}
+
+h5 {
+    font-size: 26px;
+    padding: 12px 0;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+img {
+    width: 100%;
+}
+
+a,
+a:visited,
+a:link {
+    text-decoration: none;
+    color: var(--purple);
+}
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header Section ------------------------- */
+
+.header-outer-container {
+    background: linear-gradient(to bottom right, var(--blue), var(--purple));
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: center;
+    position: relative;
+}
+
+/* Ik had onderstaande voor de achtergrond ruis op de header. */
+/* Dat werkt wel, alleen werken dan de links niet meer. */
+/* Even geen idee hoe dit op te lossen. */
+
+/*.header-outer-container::before {*/
+/*    content: "";*/
+/*    position: absolute;*/
+/*    top: 0;*/
+/*    left: 0;*/
+/*    width: 100%;*/
+/*    height: 100%;*/
+/*    background-image: url(/assets/noise-texture.png);*/
+/*    opacity: 0.6;*/
+/*    z-index: 1;*/
```

Comment:
Als je dit uitcomment, en de `z-index: 1;` verplaatst naar de `.header-outer-container` zelf, dan zou het wél moeten werken 😉

Reply:
Ik snap deze niet. Als je de background image weg haalt dat is ie sowieso niet te zien of begrijp ik jouw opmerkingen verkeerd?

Reply:
Nee het ging specifiek over de z-index. niet de gehele class

Reply:
Ik had `.header-outer-container` gezegd, maar ik bedoel `.header-inner-container`

Reply:
Dat was hem idd. Gelukt !!